addons/website/static/src/js/content/menu.js

```diff
@@ -233,4 +235,151 @@ publicWidget.registry.menuDirection = publicWidget.Widget.extend({
         $menu.addClass('dropdown-menu-' + alignment);
     },
 });
+
+publicWidget.registry.fixedHeader = publicWidget.Widget.extend({
+    selector: 'header #top_menu',
+
+    /**
+     * @override
+     */
+    start: function () {
+        self = this;
+        var menuLoading = $('o_menu_loading');
+        var header = $('.o_header_fixed');
+        if (header) {
+            if (!menuLoading) {
+                self._fixedHeader();
+            } else {
+                this.$el.one('fixedHeader', self._fixedHeader);
```

Comment:
`() => this._fixedHeader()` otherwise you have the wrong `this` in your function and you give an event parameter to your function, which you should not.